test/snippet/alphabet/aminoacid/aa10murphy.cpp

```diff
@@ -0,0 +1,48 @@
+#include <seqan3/alphabet/aminoacid/aa10murphy.hpp>
+#include <seqan3/io/stream/debug_stream.hpp>
+
+using namespace seqan3;
+
+int main()
+{
+
+{
+//! [construction]
+aa10murphy my_letter{'A'_aa10murphy};
+// doesn't work:
+// aa10murphy my_letter{'A'};
+
+my_letter.assign_char('C'); // <- this does!
+
+my_letter.assign_char('?'); // all unknown characters are converted to 'A'_aa10murphy implicitly
+
+if (my_letter.to_char() == 'A')
+    debug_stream << "yeah\n"; // "yeah";
+//! [construction]
+}
+
+{
+//! [char_literal]
+// doesn't work:
+// aa10murphy acid{'A'};
+
+// this does:
+aa10murphy acid1{'A'_aa10murphy};
+auto acid2 = 'Y'_aa10murphy; // type = aa10murphy
+//! [char_literal]
+}
+
+{
+//! [literal]
+// these don't work:
+// aa10murphy_vector foo{"ABFUYR"};
+// aa10murphy_vector bar = "ABFUYR";
+
+// but these do:
+aa10murphy_vector foo{"ABFUYR"_aa10murphy};
+aa10murphy_vector bar = "ABFUYR"_aa10murphy;
+auto bax = "ABFUYR"_aa10murphy;
+//! [literal]
+}
+
+}
```

Comment:
The general style of the examples is now to always include a full example, including main and includes so that people can copy'n'paste it. Also do not do `using namespace seqan3;` anymore, see seqan/seqan3#837

Maybe it would also be good to not copy the same example from the other, instead maybe show how an aa27 character/range is converted to reduced alphabet as that is a very typical use-case.

Reply:
> The general style of the examples is now to always include a full example, including main and includes so that people can copy'n'paste it. Also do not do `using namespace seqan3;` anymore, see #837

I removed the namespace but why should this not be a full example (besides the missing conversion example)? It has all includes and compiles

Reply:
Maybe I also misunderstand what you mean, I will anyways remove the ```using namespace``` and make a proper example of conversion that compiles and can be copy pasted

Reply:
Hm, I got your comments via mail, but not here :thinking: 

Anyway, what I meant was that the API doc should contain the entire file an not "subsections" of the file that are marked with `//! [literal]` et cetera.

Reply:
So you meant just remove the sub blocks and have a continuous file?

Reply:
Yep!